### PR TITLE
update checkstyle 4.29

### PR DESCRIPTION
**Pull request by muhlba91**

### CODE REVIEW

**Comment by muhlba91** [comment]
https://community.sonarsource.com/t/new-release-sonar-checkstyle-4-29/19425

File: checkstyle.properties [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
 
+4.29.description=Upgrade to Checkstyle 4.29
+4.29.sqVersions=[7.9,LATEST]
+4.29.date=2020-01-30
+4.29.changelogUrl=https://github.com/checkstyle/sonar-checkstyle/releases/tag/4.29
+4.29.downloadUrl=https://github.com/checkstyle/sonar-checkstyle/releases/download/4.29/checkstyle-sonar-plugin-4.29.jar
+
 4.28.description=Upgrade to Checkstyle 4.28
-4.28.sqVersions=[7.9,LATEST]
+4.28.sqVersions=[7.9,7.9*]
```
Comment (ganncamp) [inline checkstyle.properties]: You probably want to re-set the end of this range to 8.1. If you have users on this version with 8.0 or 8.1 and you end the compatibility range at 7.9, the Marketplace won't be able to notify them that there's a compatible upgrade.

Comment (muhlba91) [reply checkstyle.properties]: thank you. changed :)

**Comment by muhlba91** [comment]
@ganncamp please review again :)

i'd be super grateful if we could get this release out soon because the old version broke backwards compatibility when using JDK 8/9 in the local project. :+1:

**Comment by muhlba91** [comment]
thank you so much! :) :+1:

**Pull request merged**